packages/client-core/components/editor/inputs/ColorInput.tsx

```diff
@@ -65,16 +65,17 @@ const ColorInputPopover = (styled as any).div`
  * @constructor
  */
 
-export default function ColorInput({ value, onChange, disabled, ...rest }) {
-  const onChangePicker = useCallback(
-    ({ hex }) => {
-      onChange(new Color(hex));
-    },
-    [onChange]
-  );
+export default function ColorInput({ value, onChange, disabled,isValueAsInteger=false, ...rest }) {
+
+    const onChangePicker = useCallback(
+      ({ hex }) => {
+        onChange(isValueAsInteger?(new Color(hex)).getHex():(new Color(hex)));
+      },
+      [onChange]
+    );
 
-  //initializing hexColor by getting hexString
-  const hexColor = "#" + value.getHexString();
+    //initializing hexColor by getting hexString
+    const hexColor = "#" + (isValueAsInteger?value.toString(16):value.getHexString());
```

Comment:
Code is right but please be aware of formatting -- if you use VSCode just run the formatter before pushing. Otherwise please format these things like:
```
isValueAsInteger ? value.toString(16) : value.getHexString()
```